examples/demo/basic/contour_cmap_plot.py

```diff
@@ -62,7 +62,7 @@ def _create_plot_component():
     # Tweak some of the plot properties
     lplot.title = "Colormap and contours"
     lplot.padding = 20
-    lplot.bg_color = "white"
+    lplot.bgcolor = "white"
```

Comment:
Looks like this was just plain broken earlier. `bgcolor` is a trait on `DataView`, which `Plot` inherits from. Note that `DataView` inherits from enable `Component`, which also defines the `bgcolor` trait.